connectors/tests/test_sync.py

```diff
@@ -326,7 +325,8 @@ async def test_connector_service_poll(
     mock_responses, patch_logger, patch_ping, set_env
 ):
     await set_server_responses(mock_responses)
-    service = create_service(CONFIG)
+    load_config(CONFIG)
```

Comment:
I know `CONFIG` isn't introduced by the current PR however, having both `Config` and `CONFIG` constants confuses me. Would it be possible to rename `CONFIG` to something like `CONFIG_FILE` as it represents a full file path?

Reply:
`CONFIG_FILE` makes sense. Will update.